examples/fscloud/main.tf

```diff
@@ -103,9 +103,18 @@ module "cbr_account_level" {
     }
   }
 
-  # Demonstrates how a customized name can be set for the CBR zone
-  zone_service_ref_list = {
-    "codeengine" = "codeengine-zone-example-of-customized-zone-name"
+  # Demonstrates how a customized name and an optional location can be set for the CBR serviceRef zones
+  zone_service_ref_map = {
+    "codeengine" = {
+      zone_name           = "codeengine-zone-example-of-customized-zone-name"
+      serviceRef_location = ["au", "syd"]
```

Comment:
Is there a need to have syd (sydney) in the example if au (australia) is already there? I suspect this is redundant - so better to change the second value to another location just to avoid bringing confusion to the user.

Reply:
updated the location.